website/content/docs/commands/acl/policy/self.mdx

```diff
@@ -0,0 +1,55 @@
+---
+layout: docs
+page_title: 'nomad acl policy self command reference'
+description: >
+  The `nomad acl policy self` command fetches information about the currently set access control list (ACL) policies attached to the workload.
+---
+
+# `nomad acl policy self` command reference
+
+The `acl policy self` command is used to fetch information about the currently
+set ACL policies attached to the workload.
+
+## Usage
+
+```plaintext
+nomad acl policy self [options]
+```
+
+## General options
+
+@include 'general_options_no_namespace.mdx'
+
+## Self options
+
+- `-json` : Output the ACL policies in a JSON format.
+
+- `-t` : Format and display the ACL policies using a Go template.
+
+## Examples
+
+Fetch information about an existing ACL policies attached to the workload:
```

Comment:
```suggestion
Fetch information about an existing ACL policies attached to the workload.
```

FYI the style guide wants us to [introduce a code block with a descriptive, imperative sentence that ends with a period](https://github.com/hashicorp/tutorials/blob/main/STYLE_GUIDE.md#code-blocks).